src/fiatExchanges/types.ts

```diff
@@ -4,6 +4,6 @@ export enum SelectProviderExchangesText {
 }
 
 export enum SelectProviderExchangesLink {
-  ExchangesScreen = 'ExchangesScreen',
-  ExchangesQrScreen = 'ExchangesQrScreen',
+  ExternalExchangesScreen = 'ExternalExchangesScreen',
```

Comment:
nit: kind of strange that these have different naming conventions, but i see they match what's in statsig/line up with our screen names.